### ISSUE TITLE
use tensorflow seed

### ISSUE THREAD

**Issue by erjel** [erjel/iterative-biofilm-annotation#11]
https://github.com/erjel/iterative-biofilm-annotation/blob/148dd0985a32eac852500fda2a690488d1156cdd/iterative_biofilm_annotation/stardist/train.py#L78

tf.keras.utils.set_random_seed(
    seed
)
